sigh/lib/sigh/options.rb

```diff
@@ -121,7 +121,13 @@ def self.available_options
                                        value = value.to_s
                                        pt = %w(macos tvos ios)
                                        UI.user_error!("Unsupported platform, must be: #{pt}") unless pt.include?(value)
-                                     end)
+                                     end),
+        FastlaneCore::ConfigItem.new(key: :download_xcode_profiles,
+                                     description: "Also download Xcode managed provisioning profiles",
```

Comment:
This only applies to `download_all`, right? If so, we should mention it as such. Unfortunately this will also show up for `fastlane sigh --help`, I don't know if there is an easy way to make it only show up for `sigh download_all --help`?